includes/class-wc-ajax.php

```diff
@@ -1473,6 +1473,8 @@ public static function add_order_note() {
 			$comment_id = $order->add_order_note( $note, $is_customer_note, true );
 			$note       = wc_get_order_note( $comment_id );
 
+			do_action( 'woocommerce_add_order_note', $note, $is_customer_note );
```

Comment:
I created an action here because at the end of this function, which is called by the `wp_ajax_woocommerce_add_order_note` action, there is a call to `wp_die()` which prevents further added actions to the `wp_ajax_woocommerce_add_order_note` hook.

I'm open to suggestions here, or if creating a new action is indeed the way to go I'm not sure the best way to document it.

Reply:
I opted to go the js route instead.